hadoop-tools/hadoop-azure/src/main/java/org/apache/hadoop/fs/azurebfs/services/AbfsInputStream.java

```diff
@@ -828,9 +828,10 @@ public IOStatistics getIOStatistics() {
   @Override
   public String toString() {
     final StringBuilder sb = new StringBuilder(super.toString());
+    sb.append("AbfsInputStream@(").append(this.hashCode()).append("){");
+    sb.append("[HADOOP-18546]");
     if (streamStatistics != null) {
-      sb.append("AbfsInputStream@(").append(this.hashCode()).append("){");
-      sb.append(streamStatistics.toString());
+      sb.append(", ").append(streamStatistics);
       sb.append("}");
```

Comment:
Mehakmeet's comment is still vaild I guess. 
Shouldn't  sb.append("}");  L835    go outside of if block?

Reply:
yeah, fixing